pdns/lua-record.cc

```diff
@@ -1103,6 +1115,36 @@ static void setupLuaRecords(LuaContext& lua)
       return result;
     });
 
+  lua.writeFunction("dblookup", [](const string& record, const string& type) {
+    DNSName rec;
+    QType qtype;
+    vector<string> ret;
+    try {
+      rec = DNSName(record);
+      qtype = type;
+    }
+    catch (const std::exception& e) {
+      g_log << Logger::Error << "DB lookup cannot be performed, the name (" << record << ") or type (" << type << ") is malformed: " << e.what() << endl;
```

Comment:
```suggestion
      g_log << Logger::Error << "DB lookup cannot be performed, the name (" << record << ") or type (" << qtype << ") is malformed: " << e.what() << endl;
```

Reply:
hmm, if we reach this line, we know for a fact qtype has not been set, right?

Reply:
Hmm, right i mixed the types up.